pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -2119,3 +2141,72 @@ func getRouteHostname(obj interface{}) (host string) {
 	}
 	return host
 }
+
+func checkAndCreateAdminSecret(service kubernetes.PlatformService, namespace string, secretName string, environment api.EnvironmentType) (error, string, string) {
+	/* The default secret will be like this
+	apiVersion: v1
+	kind: Secret
+	metadata:
+	  name: kie-admin-credentials
+	type: Opaque
+	data:
+	  //adminUser
+	  username: YWRtaW4=
+	  //RedHat
+	  password: UmVkSGF0
+	*/
+
+	providedSecret, err := getSecret(service, namespace, secretName)
+
+	// The secret passed by the user doesn't exist we create one with default values
+	if err != nil {
+		username := constants.DefaultAdminUser
+		password := constants.DefaultPassword
```

Comment:
haven't we discussed to use the default password only for the trial env?

Reply:
yes, the password is overrided with a generated one except for the trial environment